### PR TITLE
Added a new variable, g:git_messenger_extra_blame_args. When set it's

### PR DESCRIPTION
**Pull request by ColinRyan**
content will be appended to the git blame command. 
Use this to add arguments to git blame (like -w).

@rhysd Let me know if there's anything I missed or anything you'd like to see changed. :)

### CODE REVIEW

**Comment by ColinRyan** [comment]
@rhysd What branch do you want this to go to?

File: autoload/gitmessenger/blame.vim [hunk 1]
```diff
@@ -397,7 +397,7 @@ let s:blame.spawn_git = funcref('s:blame__spawn_git')
 
 function! s:blame__start() dict abort
     call self.spawn_git(
-        \ ['--no-pager', 'blame', self.blame_file, '-L', self.line . ',+1', '--porcelain'],
+        \ ['--no-pager', 'blame', self.blame_file, '-L', self.line . ',+1', '--porcelain'] + split(g:git_messenger_extra_blame_args, ' '),
```
Comment (rhysd) [inline autoload/gitmessenger/blame.vim]: Please also apply this change to below line.

https://github.com/rhysd/git-messenger.vim/blob/72975c9be9267689cd408a5ea6a02200502217d0/autoload/gitmessenger/blame.vim#L42

The line also runs `blame` for getting the older change. Please take care that the extra arguments must be inserted before `--` option in this case.

**Comment by rhysd** [comment]
> What branch do you want this to go to?

Current setting is perfect. I want to merge this into master branch.

File: autoload/gitmessenger/blame.vim (hunk 1 above)
Comment (ColinRyan) [reply autoload/gitmessenger/blame.vim]: no sweat @rhysd

Comment (ColinRyan) [reply autoload/gitmessenger/blame.vim]: I've updated the code as per your request.

**Pull request merged**